client/components/section-nav/style.scss

```diff
@@ -298,7 +302,8 @@
 
 	.notouch & {
 		&:hover {
-			color: var( --color-accent );
+			background-color: var( --color-blue-0 );
```

Comment:
What is the reasoning behind introducing `--color-blue-X` type color variables? Wouldn't we want to prevent the use of any color names and use a more semantic representation instead like we did with `--color-primary` etc?

Reply:
It seems we need more shades to also cover only slight variations, see https://github.com/Automattic/wp-calypso/pull/29481#issuecomment-448070958. Agreed that we shouldn't use it directly though - will follow up with a patch.